metacat-connector-hive/src/main/java/com/netflix/metacat/connector/hive/HiveConnectorFactory.java

```diff
@@ -90,10 +95,20 @@ public HiveConnectorFactory(
             throw new IllegalArgumentException(
                 String.format("Failed creating the hive metastore client for catalog: %s", catalogName), e);
         }
-        final Module hiveModule = new HiveConnectorModule(config, catalogName, configuration, infoConverter, client);
+        final Module hiveModule = new HiveConnectorModule(config,
+            catalogName, configuration, infoConverter, client, registry);
         this.injector = Guice.createInjector(hiveModule);
     }
 
+    private IMetacatHiveClient createLocalClient() throws Exception {
+        final HiveConf conf = getDefaultConf();
+        configuration.forEach(conf::set);
+        //TO DO Change the usage of DataSourceManager later
```

Comment:
really minor but you should make this TODO one word so we can find it quickly and IDE's will pick it up